python/setup.py

```diff
@@ -0,0 +1,34 @@
+from setuptools import setup, find_packages
+
+from setuptools import Extension
+from setuptools.command.build_ext import build_ext
+import sys
+import setuptools
+
+import glob
+import os
+
+ext_modules = [
+    Extension(
+        'libsunbeam',
+        glob.glob(os.path.join('src', '*.cpp')),
```

Comment:
We generally do not use `glob` in opm - but rather list the source files explicitly.